### PR TITLE
Version/3.29

### PR DESCRIPTION
**Pull request by lbudai**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -111,6 +143,4 @@ of syslog-ng, contribute.
 
 We would like to thank the following people for their contribution:
 
-Airbus Commercial Aircraft, Andras Mitzki, Antal Nemes, Attila Szakacs,
-Balazs Scheidler, Gabor Nagy, Laszlo Budai, Laszlo Szemere, László Várady,
-Péter Kókai, Vatsal Sisodiya, Vivin Peris.
+<Fill this by the internal news file creating tool>
```
Comment (MrAnno) [inline NEWS.md:146]: 0140454, Andras Mitzki, Antal Nemes, Attila Szakacs, Balazs Scheidler, Christian Tramnitz, chunmeng, Gabor Nagy, Laszlo Budai, Laszlo Szemere, László Várady, MileK, Norbert Takacs, Peter Czanik, Péter Kókai, Terez Nemes.

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: NEWS.md [hunk 2]
```diff
@@ -1,104 +1,136 @@
...
-
-   Example:
-   ```
-   log {
-      source { system(); };
-      destination { http( url("SYSLOG_SERVER_IP:PORT") proxy("PROXY_IP:PORT") method("POST") ); };
-   };
-   ```
-   ([#3253](https://github.com/syslog-ng/syslog-ng/pull/3253))
+<Fill this block manually from the blocks below>
```
Comment (MrAnno) [inline NEWS.md:6]: Do you want to add highlights for this release? `panos-parser()` or PCRE dupnames might be interesting.

Comment (lbudai) [reply NEWS.md:6]: could you open a sub-pr for this? meanwhile I'm working on the release tarball & packages.

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: NEWS.md (hunk 2 above)
Comment (MrAnno) [reply NEWS.md:6]: Yeah, I'm going to open a PR against your fork. :)

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**